api/cluster.go

```diff
@@ -0,0 +1,29 @@
+package api
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/raintank/metrictank/api/middleware"
+	"github.com/raintank/metrictank/api/models"
+	"github.com/raintank/metrictank/api/rbody"
+	"github.com/raintank/metrictank/cluster"
+)
+
+func (s *Server) getNodeStatus(ctx *middleware.Context) {
+	rbody.WriteResponse(ctx, rbody.NewJsonResponse(200, cluster.ThisNode, ""))
```

Comment:
no it is not.  This is a good catch.  I wonder how many other places i have not considered thread safety when marshaling structs to json.